src/gretel_synthetics/train.py

```diff
@@ -24,41 +28,71 @@
     level=logging.INFO)
 
 
+class LossHistory(tf.keras.callbacks.Callback):
+
+    def __init__(self):
+        self.losses = []
+        self.accuracy = []
+
+    def on_epoch_end(self, epoch, logs={}):
+        self.losses.append(logs.get('loss'))
+        self.accuracy.append(logs.get('accuracy'))
+
+
+def save_model_params(store: BaseConfig):
+    save_path = Path(store.checkpoint_dir) / 'model_params.json'
+    logging.info(f"Saving model history to {save_path.name}")
+    with open(save_path, 'w') as f:
+        json.dump(store.as_dict(), f, indent=2)
+
+
+def save_history_csv(history:LossHistory, store:BaseConfig):
```

Comment:
Should these be methods on the config class so they can be called directly from the store instance?

Reply:
1. For save model params, yes. Will refactor.
2. For saving the model training history (loss/accuracy), I think it makes sense to keep it in train.py